.github/workflows/test.yml

```diff
@@ -146,7 +146,7 @@ jobs:
 
         run: cabal test wrapper-test --test-options="$TEST_OPTS --rerun-log-file .tasty-rerun-log-wrapper" || cabal test wrapper-test --test-options="$TEST_OPTS  --rerun-log-file .tasty-rerun-log-wrapper" || cabal test wrapper-test --test-options="$TEST_OPTS  --rerun-log-file .tasty-rerun-log-wrapper"
 
-      - if: matrix.test && matrix.ghc != '9.0.1' && matrix.ghc != '9.2.1'
+      - if: matrix.test && matrix.ghc != '9.2.1'
```

Comment:
FYI @jneira I conservatively merged to allow 9.0.1 since we fixed it on master but not allow 9.2.1, but maybe fixing it for 9.0.1 also fixed it for 9.2.1. Not sure, we should try after this is merged perhaps.